lib/lotus/view/rendering/partial_finder.rb

```diff
@@ -34,14 +35,24 @@ class PartialFinder < TemplateFinder
         # @since 0.4.3
         # @api private
         def find
-          if path = partial_template_under_view_path
-            View::Template.new(path, @view.configuration.default_encoding)
-          else
-            super
-          end
+          find_cached_template
         end
 
         protected
+
+        # @since x.x.x
+        # @api private
+        def find_cached_template
+          conf = Lotus::View::Configuration.for(@view)
```

Comment:
@stevehook I see and sorry for suggesting a wrong solution. If this works and not the other one, we should use your solution. Thank you :+1: 

One last thing before to merge. Because the code has changed since the last benchmark, can we run it again and compare? Thanks :100: :sparkling_heart: